pkg-manager/resolve-dependencies/src/index.ts

```diff
@@ -319,20 +321,116 @@ function verifyPatches (
   })
 }
 
+interface RegistryFileFields {
+  size: number
+  type: 'File'
+  path: string
+  contentType: string
+  hex: string
+  isBinary: boolean
+  linesCount: number
+}
+
+interface RegistryFileList {
+  files: {
+    [path: string]: RegistryFileFields
+  }
+}
+
+async function fetchRegistryFS (pkgName: string, pkgVer: string): Promise<RegistryFileList> {
```

Comment:
I think this would be a better name
```suggestion
async function fetchPkgIndex (pkgName: string, pkgVer: string): Promise<RegistryFileList> {
```

Reply:
also, don't user Ver for Version.